docs/builder-guides/XVM and WASM/interact_with_wasm_contract.md

```diff
@@ -0,0 +1,318 @@
+---
+sidebar_position: 3
+---
+
+# Interact with WASM contract with astar.js
+## TL;DR
+During this guide, we'll be guiding you on how to interact with WASM contract with astar.js
+
+---
+
+## What is a astar.js?
+Astar.js library provides application developers the ability to query a node and interact with the astar/shiden/shibuya chains using Javascript/Typescript.
+
+## Packages
+The @polkadot/api-contract packages is to be used alongside the @astar-network/astar-api package. With that in mind, we can install from npm, so we are not going to waste too much time with the bare basics.
+
+yarn add @astar-network/astar-api@beta @polkadot/api @polkadot/api-contract
+
+## API connection
+
+The API provides application developers the ability to query astar node for sending transactions to contract.
```

Comment:
Maybe it's me, but "query" does not work with "send transaction". "Query data" vs "order to send a transaction". IMO, just "ability to send transaction" would be enough.